### ISSUE TITLE
Whoops, looks like something went wrong.

### ISSUE THREAD

**Issue by yuri25** [CodepadME/laravel-tricks#66]
The installation was successful. When you open the site - this error: Whoops, looks like something went wrong.

 Predis\Connection\ConnectionException
…/­vendor/­predis/­predis/­lib/­Predis/­Connection/­AbstractConnection.php141

Predis \ Connection \ ConnectionException
Connection refused [tcp://127.0.0.1:6379]
- Helper method to handle connection errors.
   *
  - @param string $message Error message.
  - @param int    $code    Error code.
    */
    protected function onConnectionError($message, $code = null)
    {
    CommunicationException::handle(new ConnectionException($this, "$message [{$this->parameters->scheme}://{$this->getIdentifier()}]", $code));
    }

**Comment by koppenhoefer** [comment]
In case anyone, like me, is trying to solve this using Docker containers on a Mac,.. and using Laradock. Here are the edits I needed to get this 'Whoops error'. :-)

1. Ensure that your php-fpm container has a defined hostname.
I edited laradoc/docker-compose.yml to add the following line to the php-fpm container definition.
hostname: php-fpm

2. Edit bootstrap/start.php detectEnvironment section to use that hostname.
I did this: 'local' => array('php-fpm'),

3. Edit main.blade.php to change css/laratricks.min.4.css to be css/laratricks.min.css 
4. Edit navigation.blade.php to fix logo@2x.1.png to read logo@2x.png.

After those edits, I brought my containers down,.. and then back up again. Everything worked fine. Good luck. - shawn
